sonar-html-plugin/src/main/resources/org/sonar/l10n/web/rules/Web/S5258.html

```diff
@@ -0,0 +1,36 @@
+<p>Tables used for layout should not include semantic markup, such as <code>&lt;th&gt;</code> elements, as it can confuses assistive technologies. At
+best this information will be ignored by screen readers and makes the code less maintainable. However it can also confuse some screen readers and
+reduce the web page accessibility.</p>
+<p>This rule raises an issue when a <code>&lt;table&gt;</code> element containing the&nbsp;<code>role</code> attribute set to
+<code>"presentation"</code> or <code>"none"</code> also contains any of:</p>
+<p> * a <code>&lt;caption&gt;</code> element</p>
+<p> * a <code>&lt;th&gt;</code> element</p>
+<p> * a non-empty <code>summary</code> attribute</p>
+<p> * an <code>&lt;td&gt;</code> element with a <code>headers</code> or <code>scope</code> attribute</p>
```

Comment:
Can you please regenerate rules metadata. I updated jira ticket description so that markup will be correctly generated. Thanks